src/dockleHelper.ts

```diff
@@ -34,6 +37,58 @@ export async function getDockle(): Promise<string> {
     return dockleToolPath;
 }
 
+export function isCisChecksEnabled(): boolean {
+    return inputHelper.addCISChecks.toLowerCase() === "true";

```

Comment:
NIT: You could use `JSON.parse` instead as well.

![image](https://user-images.githubusercontent.com/25762922/77253545-fd545e80-6c80-11ea-8400-e956bb8ff0f5.png)

Reply:
Or move this logic to the helper itself

Reply:
> NIT: You could use `JSON.parse` instead as well.
> 
> ![image](https://user-images.githubusercontent.com/25762922/77253545-fd545e80-6c80-11ea-8400-e956bb8ff0f5.png)

Won't that be an overkill?

Reply:
Moved to inputHelper.